src/shared/ui/main-title/main-title.tsx

```diff
@@ -0,0 +1,21 @@
+import { VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
+
+import styles from './main-title.module.scss';
+
+type MainTitleProps = React.HTMLAttributes<HTMLHeadingElement> & VariantProps<typeof mainTitleVariants>;
```

Comment:
Try to avoid using `React.SomeType` import just `SomeType` itself

Reply:
I used an example from the library and didn't pay attention